index.html

```diff
@@ -55,8 +60,106 @@ <h1>I’m Nyamekye.
+                        <ul class="lang">
+                            <li><a href="#">HTML</a></li>
+                            <li><a href="#">CSS</a></li>
+                            <li><a href="#">Javascript</a></li>
+                        </ul>
+                        <button class="btn-default">See project</button>
+                    </div>
+                    <div class="card-left-desk">
+                        <img src="images/work/Snapshoot Portfolio(1).png" alt="snap-card">
+                    </div>
+                </div>
 
+                <div class="card">
+                    <div class="card-left">
+                        <img src="images/work/Snapshoot Portfolio(2).png" alt="snap-card">
+                    </div>
+                    <div class="card-right">
+                        <h2>Tonic</h2>
+                        <ul class="cano">
+                            <li>CANOPY<i class="fa fa-circle"></i></li>
+                            <li>Back End Dev<i class="fa fa-circle"></i></li>
+                            <li>2015</li>
+                        </ul>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <ul class="lang">
+                            <li><a href="#">HTML</a></li>
+                            <li><a href="#">CSS</a></li>
+                            <li><a href="#">Javascript</a></li>
+                        </ul>
+                        <button class="btn-default">See project</button>
+                    </div>
+                </div>
+
+                <div class="card">
+                    <div class="card-left-mobile">
+                        <img src="images/work/Snapshoot Portfolio(3).png" alt="snap-card">
+                    </div>
+                    <div class="card-right">
+                        <h2>Multi-Post
+                            Stories</h2>
+                        <ul class="cano">
+                            <li>CANOPY<i class="fa fa-circle"></i></li>
+                            <li>Back End Dev<i class="fa fa-circle"></i></li>
+                            <li>2015</li>
+                        </ul>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <ul class="lang">
+                            <li><a href="#">HTML</a></li>
+                            <li><a href="#">CSS</a></li>
+                            <li><a href="#">Javascript</a></li>
+                        </ul>
+                        <button class="btn-default">See project</button>
+                    </div>
+                    <div class="card-left-desk">
+                        <img src="images/work/Snapshoot Portfolio(3).png" alt="snap-card">
+                    </div>
+                </div>
+            </div>
+        </section>
+        <!-- end of works -->
```

Comment:
- [ ] Kindly note that the `About` section is part of this milestone. I highly suggest you work on it. Please also make sure to align the contents of the About section with flexbox. 🤝️

<img src="https://github.com/evansnyamekye/Post-index/assets/17243325/ad274d28-32bd-4b52-ad88-b6f849ccd106" width="300">